app/views/declaration/additional_fiscal_references.scala.html

```diff
@@ -43,6 +43,38 @@
 
 @(itemId:String, form: Form[AdditionalFiscalReference], additionalReferences: Seq[AdditionalFiscalReference] = Seq.empty)(implicit request: Request[_], messages: Messages)
 
+@referenceTable = {
```

Comment:
Why you're not using `removable_elements_table` from components? We should have one approach to show this part of the page

Reply:
Now it continas submits instead fo buttons - i will try think about unification